source/_integrations/music_assistant.markdown

```diff
@@ -23,53 +23,35 @@ There is currently support for the following Home Assistant Platforms:
 
 All of the Home Assistant [Media Player Control Actions](https://www.home-assistant.io/integrations/media_player/#media-control-actions) are supported.
 
-{% include integrations/config_flow.md %}
-
-### Manual configuration
-
-Under normal circumstances, Home Assistant automatically discovers your running Music Assistant Server. If there is something special about the HA or MA setup (for example, the MA server is running as a remote Docker container) or discovery is not working, you can manually specify the URL to your Music Assistant server. If the Music Assistant Server is not installed then follow these [installation instructions](https://music-assistant.io/installation/).
-
-## Media player
+The `media_content_id` payload for `media_player.play_media` can be any of the following:
 
-The Music Assistant integration creates media player entities for all players available in MA including those imported from Home Assistant. This is needed to provide the full functionality Music Assistant has to offer. These entities will display media information, playback progress, and playback controls.
+- The name of a track, artist or album. (for example, `Queen`)
+- A track or album combined with the artist name (for example, `Queen - Innuendo`)
+- A streaming provider URI (for example, `spotify://artist/12345`)
 
-### Action `media_player.play_media`
+The `media_content_id` payload for `media_player.browse_media` must be a URI of the form `library://artist/1`, `library://album/20`, or `spotify://album/5zj4Ej0FrlJQaSo0d6cttH`. The type of item that the URI refers to must be an album or artist.
 
-Play media hosted on a Music Assistant server on a Music Assistant player. The action configuration is as described in the [media player documentation](https://www.home-assistant.io/integrations/media_player/#action-media_playerplay_media)
+These URIs can be obtained from, for example, the output of the `get_library` or `search` actions described below or the `media_player.browse_media` HA action. 
 
-The `media_content_id` payload can be any of the following:
+{% include integrations/config_flow.md %}
 
-- The name of a track, artist or album. (for example, "Queen")
-- A track or album combined with the artist name (for example, "Queen - Innuendo")
-- A streaming provider URI (for example, `spotify://artist/12345`)
+### Manual configuration
 
-#### Examples
+Under normal circumstances, Home Assistant automatically discovers your running Music Assistant Server. If there is something special about the HA or MA setup (for example, the MA server is running as a remote Docker container) or discovery is not working, you can manually specify the URL to your Music Assistant server. If the Music Assistant Server is not installed, then follow these [installation instructions](https://music-assistant.io/installation/).
 
-Play Adele's album 25
+## Removing the integration
```

Comment:
I would move the removal instructions to the end of the page, as it marks the end of the integration's life-cycle.